### PR TITLE
Cosmetic fixes to pass markdown linting.

### PR DESCRIPTION
**Pull request by paulczar**
No functional change, just removing all the lint errors
given by VS Code markdown linting.

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

### CODE REVIEW

File: docs/Upgrade.md [hunk 1]
```diff
@@ -1,48 +1,64 @@
...
+
+    Find the secret whose name ends with `-harbor-ingress` (expose service via `Ingress`) or `-harbor-nginx`(expose service via `ClusterIP` or `NodePort`)
+
+2) Export the secret as yaml file:
+
+    ```bash
+    kubectl get secret secret-name -o yaml > secret.yaml
+    ```
+
+3) Replace the `secret-name` in `secret.yaml` with the one from step **1**
```
Comment (ywk253100) [inline docs/Upgrade.md]: It isn't a separate step, it should be merge into step 2).

Comment (paulczar) [reply docs/Upgrade.md]: done!

File: docs/Upgrade.md [hunk 2]
```diff
@@ -1,48 +1,65 @@
...
+2) Export the secret as yaml file:
+
+
+    ```bash
+    kubectl get secret <secret-name-from-step-1> -o yaml > secret.yaml
+    ```
+
+
+
+4) Rename the secret by setting `metadata.name` in `secret.yaml`
```
Comment (ywk253100) [inline docs/Upgrade.md]: Should be 3,4,5 rather than 4,5,6

Comment (paulczar) [reply docs/Upgrade.md]: hah! good catch.  fixed.

**Pull request merged**